dspace-server-webapp/src/test/java/org/dspace/app/rest/BundleRestRepositoryIT.java

```diff
@@ -636,4 +641,49 @@ public void deleteBundle_NoAuthToken() throws Exception {
                 .andExpect(status().isOk());
     }
 
+    @Test
+    public void getEmbeddedItemForBundle() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        bundle1 = BundleBuilder.createBundle(context, item)
+                .withName("testname")
+                .build();
+
+        context.restoreAuthSystemState();
+
+        getClient().perform(get("/api/core/bundles/" + bundle1.getID() + "?embed=item"))
+                .andExpect(status().isOk())
+                .andExpect(content().contentType(contentType))
+                .andExpect(jsonPath("$._embedded.item",
+                        ItemMatcher.matchItemWithTitleAndDateIssued(item, "Public item 1", "2017-10-17")
+                ));
+    }
+
+    @Test
+    public void linksToFirstItemWhenMultipleItems() throws Exception {
```

Comment:
I would prefer to remove this test as this is an use case that we don't want to support

Reply:
Again, as before, I'm OK with this test, as it proves our REST API won't completely error out if somehow a site has a Bundle belonging to two Items.

That said, I agree with @abollini  that this isn't something we should actively support.  So, again, let's add some JavaDocs/comments to this test method to say something like:
`This test proves that, if a bundle is linked to multiple items, we only ever return the first item. **NOTE: DSpace does NOT support or expect to have a bundle linked to multiple items**. But, because the database does allow for it, this test simply proves the REST API will respond without an error.`